### PR TITLE
SR1.1.1. Add support for showing daily standard deviation on inflammation data

### PR DESCRIPTION
**Pull request by fionaroman99**
Extend the graphs to show the daily standard deviation as required by SR1.1.1.

### CODE REVIEW

File: inflammation-analysis.py [hunk 1]
```diff
@@ -22,10 +22,14 @@ def main(args):
     for filename in in_files:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {
```
Comment (l-m-thole) [inline inflammation-analysis.py:25]: suggestion: move curly bracket at end to next line

File: inflammation-analysis.py [hunk 2]
```diff
@@ -22,10 +22,14 @@ def main(args):
...
-            'average': models.daily_mean(inflammation_data), 
-            'max': models.daily_max(inflammation_data),
-            'min': models.daily_min(inflammation_data)}
+# <<<<<<< feature-std-dev
```
Comment (l-m-thole) [inline inflammation-analysis.py:25]: I don't understand what this line does

File: inflammation/models.py [hunk 3]
```diff
@@ -41,3 +41,13 @@ def daily_min(data):
     :return: Numpy array with the minimum daily value."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (l-m-thole) [inline inflammation/models.py:45]: change s_dev to standard_deviation or std_dev

File: tests/test_models.py [hunk 4]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
```
Comment (l-m-thole) [inline tests/test_models.py:36]: are these random numbers?

File: inflammation/models.py [hunk 5]
```diff
@@ -41,3 +41,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```
Comment (l-m-thole) [inline inflammation/models.py:52]: heuy, you could use the function np.std instead of calculating the standard edviation from scratch :)

**Review by l-m-thole — CHANGES_REQUESTED** [review]
looks good, but consider adding a few more tests

File: tests/test_models.py (hunk 4 above)
Comment (fionaroman99) [reply tests/test_models.py:36]: these are the numbers that were given in the exercise i think

**Pull request merged**